packages/import-maps-generate/README.md

```diff
@@ -77,7 +94,11 @@ you can archive that via an override in your `package.json`
 }
 ```
 
-which will result in the following import map.
+::: warning
+Note that if you apply overrides, you may also need to specify deletions for the generated scope in the importmap.
```

Comment:
you will need to override the `directory` map right? these are not to be confused with scopes

```
imports: {
   'foo/', '/path/to/foo/', // "directory map" (no clue if there is a name for it 🙈 )
}
scopes: {
   '/path/to/script.js': { // scope to provide a specific import map (which overrides) for a specific path/url
     'foo/': '/path/to/fork/
  }
}
```

Reply:
Lets go with directory map for now yeah :)

Reply:
It's just called a `package` mapping: https://github.com/WICG/import-maps#packages-via-trailing-slashes